packages/code-snippet/style/index.css

```
@@ -58,48 +72,63 @@
 }
 
 .elyra-codeSnippet-item {
-  display: inline-flex;
-  width: 100%;
-  justify-content: space-between;
-  padding-top: 5px;
-  padding-bottom: 5px;
+  border-bottom: 1px solid #eeeeee;
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  padding: 0;
+}
+
+.elyra-expandableContainer-title {
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  padding: 0px 4px;
+  height: 36px;
+}
+
+.elyra-expandableContainer-title:hover {
+  background-color: #eeeeee;
 }
 
 .elyra-expandableContainer-name {
-  padding-bottom: 5px;
-  display: inline-flex;
+  flex-grow: 1;
+  font-size: 13px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  padding: 4px 0 4px 2px;
```

```suggestion
  flex-grow: 1;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
  padding: 4px 0 4px 2px;
  line-height: 28px;
```
Based on your updates to the onClick this span should vertically fill the parent (with its height of 36px) so the onClick is on the whole div and not just the text. Also I'm not sure why you shrunk the font size from the default to 13px

---

the only reason for changing the font size was to match the rest of the font sizes in the other panels (i.e., file browser) but looking closely those are more accurately set to `var(--jp-ui-font-size1)` (which equates to 13px) perhaps we should set these font to `var(--jp-ui-font-size1)` as well?

---

if you use a var like that then I'm ok with changing the font size